zebra-rpc/src/methods.rs

```diff
@@ -3210,17 +3220,19 @@ impl GetBlockchainInfoResponse {
 /// This is used for the input parameter of [`RpcServer::get_address_balance`],
 /// [`RpcServer::get_address_tx_ids`] and [`RpcServer::get_address_utxos`].
 #[derive(Clone, Debug, Eq, PartialEq, Hash, serde::Deserialize, serde::Serialize)]
-#[serde(from = "DAddressStrings")]
-pub struct AddressStrings {
+#[serde(from = "DGetAddressBalanceRequest")]
+pub struct GetAddressBalanceRequest {
```

Comment:
We should update the docs for `GetAddressBalanceRequest` to remove the references to `RpcServer::get_address_tx_ids` and `RpcServer::get_address_utxos` now that those have their own request types.